### PR TITLE
FORMIT-10996 Fixes Set Location plugin import not showing new location

### PR DESCRIPTION
**Pull request by brett-garrison-adsk**
Uses "Move Location to Origin" checkbox to distinguish whether user is setting origin relative to old location or setting new location

Show error notification if map coordinates don't return from Bing geocoding search results

Address not used if location image is deleted

Setting location after relative origin change shows no world center

Zooming and locate me map controls don't update import view

### CODE REVIEW

File: v20_0_prerelease/location.js [hunk 1]
```diff
@@ -78,14 +82,9 @@ class FormItMap {
                             this._updatePushPin();
                             this._focusLocation();
                         });
-                        //no results found by auto-suggest API, so try to geocode with exactly what the user gave us.
                     } else {
-                        this._address = this._addressInput.value;
-                        this._location = undefined;
-                        this._geocodeLocationAddress(() => {
-                            this._updatePushPin();
-                            this._focusLocation();
```
Comment (matturday) [inline v20_0_prerelease/location.js]: @garrisb82 can you describe the change here so @deanstein can understand the UX consequences? In this case we won't update the push pin or focus the location?

File: v20_0_prerelease/location.js [hunk 2]
```diff
@@ -329,6 +340,25 @@ class FormItMap {
         this._deselectStation();
     }
 
+    useLastLocation() {
```
Comment (matturday) [inline v20_0_prerelease/location.js]: Naming of this is confusing. This uses the location from the file, so I'd name it that way. Perhaps syncLocationFromFile

**Review by matturday — APPROVED** [review]
Just a shallow review here. It's difficult to understand some of the implications of these changes in the context of the previous code. I know the previous code isn't the most intuitive. Some comments on the updates would be helpful. 

The code updates look fine though, one minor comment.

File: v20_0_prerelease/location.js (hunk 1 above)
Comment (josh-goldstein-adsk) [reply v20_0_prerelease/location.js]: Brett has demo'd the changes for me and Helena and we liked what we saw.

Comment (brett-garrison-adsk) [reply v20_0_prerelease/location.js]: @matturday the push pin still updates, but that is done in the _setAddress call this was all replaced with.

**Review by josh-goldstein-adsk — APPROVED** [review]

**Pull request merged**